plugins/modules/aws_glue_crawler.py

```diff
@@ -0,0 +1,409 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: aws_glue_crawler
+version_added: 1.5.0
+short_description: Manage an AWS Glue crawler
+description:
+    - Manage an AWS Glue crawler. See U(https://aws.amazon.com/glue/) for details.
+requirements: [ boto3 ]
+author:
+  - 'Ivan Chekaldin (@ichekaldin)'
+options:
+  database_name:
+    description:
+      - The name of the database where results are written.
+    type: str
+  description:
+    description:
+      - Description of the crawler being defined.
+    type: str
+  name:
+    description:
+      - The name you assign to this crawler definition. It must be unique in your account.
+    required: true
+    type: str
+  recrawl_policy:
+    description:
+      - A policy that specifies whether to crawl the entire dataset again, or to crawl only folders that were added since the last crawler run.
+      - I(RecrawlBehavior) specifies whether to crawl the entire dataset again or to crawl only folders that were added since the last crawler run
+      - Options are CRAWL_EVERYTHING or CRAWL_NEW_FOLDERS_ONLY.
+    type: dict
+  role:
+    description:
+      - The name or ARN of the IAM role associated with this crawler.
+      - Required when I(state=present).
+    type: str
+  schema_change_policy:
+    description:
+      - The policy for the crawler's update and deletion behavior.
+      - I(UpdateBehavior) defines the update behavior when the crawler finds a changed schema. Options are LOG or UPDATE_IN_DATABASE.
+      - I(DeleteBehavior) defines the deletion behavior when the crawler finds a deleted object. Options are LOG, DELETE_FROM_DATABASE, DEPRECATE_IN_DATABASE
+    type: dict
+  state:
+    description:
+      - Create or delete the AWS Glue crawler.
+    required: true
+    choices: [ 'present', 'absent' ]
+    type: str
+  table_prefix:
+    description:
+      - The table prefix used for catalog tables that are created.
+    type: str
+  tags:
+    description:
+      - A hash/dictionary of tags to be applied to the crawler.
+      - Remove completely or specify an empty dictionary to remove all tags.
+    default: {}
+    type: dict
+  targets:
+    description:
+      - A list of targets to crawl. See example below.
+      - Required when I(state=present).
+    type: dict
+extends_documentation_fragment:
+- amazon.aws.aws
+- amazon.aws.ec2
+
+'''
+
+EXAMPLES = r'''
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+
+# Create an AWS Glue crawler
+- community.aws.aws_glue_crawler:
+    name: my-glue-crawler
+    database_name: my_database
+    role: my-iam-role
+    schema_change_policy:
+      DeleteBehavior: DELETE_FROM_DATABASE
+      UpdateBehavior: UPDATE_IN_DATABASE
+    recrawl_policy:
+      RecrawlBehavior: CRAWL_EVERYTHING
```

Comment:
Your current version isn't very ansible/python-like

```suggestion
    schema_change_policy:
      delete_behavior: DELETE_FROM_DATABASE
      update_behavior: UPDATE_IN_DATABASE
    recrawl_policy:
      recrawl_behavior: CRAWL_EVERYTHING
```

What you can do is use snake_dict_to_camel_dict before handing this to AWS

Reply:
Thanks, I replaced this with:

```
    schema_change_policy:
      delete_behavior: DELETE_FROM_DATABASE
      update_behavior: UPDATE_IN_DATABASE
    recrawl_policy:
      recrawl_ehavior: CRAWL_EVERYTHING
```